### ISSUE TITLE
Miss alt attribute for feedback

### ISSUE THREAD

**Issue by lyndaidaii** [NuGet/NuGetGallery#8224]
Title: WCAG 1.1.1: Ensures <img> elements have alternate text or a role of none or presentation (img)
Tags: Accessibility, WCAG 1.1.1, image-alt

Issue: Ensures <img> elements have alternate text or a role of none or presentation (image-alt - https://dequeuniversity.com/rules/axe/4.0/image-alt?application=msftAI)

Target application: NuGet Gallery | Home - https://www.nuget.org/

Element path: iframe[src=""];img

Snippet: <img src="https://d6tizftlrpuof.cloudfront.net/themes/production/microsoft-nuget-button-5e3ed075d5d2d6d1f91dfb8fd5e15d01.png" width="40" height="130">

How to fix: 
Fix any of the following:
  Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute or the title attribute is empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"

Environment: Chrome version 85.0.4183.121

====

This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
